queries.sql

```diff
@@ -43,3 +43,96 @@ FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. 
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+
+-- Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+SELECT * FROM animals; -- species column = unspecified
+ROLLBACK;
+
+--Inside a transaction: Update the animals table by setting the species column to digimon for all animals that have a name ending in mon. Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+
+BEGIN;
+
+-- Update species to 'digimon' for animals with names ending in "mon"
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+-- Update species to 'pokemon' for animals with no species set
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
+-- Inside a transaction delete all records in the animals table, then roll back the transaction.
+
+BEGIN;
+
+-- Delete all records in the animals table
+DELETE FROM animals;
+
+-- Roll back the transaction to undo the deletion
+ROLLBACK;
```

Comment:
- [ ] 👍 Good work on updating the screenshot. Now, as the previous reviewer mentioned to maintain consistency and provide a clear overview of your transaction process, kindly update the `queries.sql` file to include the full transaction, just as you executed it. This file should reflect all the SQL statements of your transaction in the order they were executed, including the `SELECT` query to verify that changes persist after the commit. 

By updating the `queries.sql` file, you will ensure that your project documentation aligns with the executed code, making it easier for reviewers to understand your work. Keep up the great effort! 👍📝📊